tests/ocf-meta-inf_manifest/EPUB/content.xml

```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Similar to your other PR, please rename the files (and the test ID in the package file). I suggest ocf-metainf-manifest.

Reply:
thnx. updated